### PR TITLE
Homework # 4 cars_lot.rb

### PR DESCRIPTION
**Pull request by djuare**

### CODE REVIEW

File: DavidJuarez-djuare/hw_4/cars_lot.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+toyota = {brand: "toyota", model: "prius", year: 2012 , price: 39000 , is_electric: true }
+bmw = {brand: "bmw", model: "bmw 3 Series 328i ", year: 2015 , price: 51575 , is_electric: false }
+audi = {brand: "audi", model: "A8 Sedan ", year: 2015 , price: 77400 , is_electric: false }
+
+cars = []
+cars.push(tesla, porsche, toyota, bmw, audi)
+show_all_cars(cars)
+
+#add_cars(cars)
+
```
Comment (chylaudes) [inline DavidJuarez-djuare/hw_4/cars_lot.rb]: Try invoking the add_cars method:
```
add_cars(cars, tesla, porsche, toyota, bmw, audi)
```
see line 39

File: DavidJuarez-djuare/hw_4/cars_lot.rb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            puts"#{car[:brand]} REDUCES POLLUTION AND IT IS A TOYOTA CAR \n"
+        end
+
+    end
+
+end
+
+#write a method that accepts an unlimited number of cars (or arguments)
+def add_cars(*all_cars)
+
```
Comment (chylaudes) [inline DavidJuarez-djuare/hw_4/cars_lot.rb]:
```
add_cars(array, *all_cars)
  all_cars.each do |car|
    array.push(car)
  end
end

```

**Comment by chylaudes** [comment]
Great job!  I see your attention to detail in spacing and everything.  It seems like you understand the .each method very well

**Comment by djuare** [comment]
Hi Christine,

Thanks for your comments. I improved my program including the extra bonus. I will push it in a little while.

Kind regards,
David Juarez

**Pull request merged**